cohort-engine/src/main/java/com/ibm/cohort/engine/helpers/CanonicalHelper.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * (C) Copyright IBM Corp. 2021
```

Comment:
nitpick: copyright template has 2 years

Reply:
My understanding was that files should be dated with the year that they were created and then updated. For all new files, I've been using the 2021-only format. Happy to change if that isn't right. Do we have a copyright template stored somewhere as reference?

Reply:
I was going off https://test.cloud.ibm.com/docs/healthcare-framework?topic=healthcare-framework-3-1-prepare-source-code - it doesn't really call out the scenario of when the start and update are the same year, but I think in most other classes it's 2 dates